daprdocs/layouts/partials/components/secret-stores.html

```diff
@@ -19,13 +19,15 @@
     <h3>{{ $group }}</h3>
     <table>
         <tr>
+            <th>Multiple Key-Values Per Secret</th>
```

Comment:
This column should be between component and status, just like state stores

Reply:
Will send a review to fix this in a bit.